### ISSUE TITLE
Initializing weights and biases with stddev=0.1 seems to ease convergence

### ISSUE THREAD

**Issue by reuben** [mozilla/DeepSpeech#138]
On my machine with the LDC93S1 dataset, initializing weights and biases with stddev=0.1 eliminates the CTC "path not found" warnings and makes us successfully overfit the data, even without the changes from issue #109.

I haven't tested with other datasets yet, but I'm filing this issue so we can investigate.

**Comment by kdavis-mozilla** [comment]
At most this will work on the current master for LDC93S1 as other data sets can not present the network more than the first batch without #109 being addressed. LDC93S1 as it stands has only one batch in which all elements are equal. So, it's a special case.

I guess this likely suggest simply that stddev=0.1 is a good value for the initialization hyperparameter.

**Comment by kdavis-mozilla** [comment]
Also what seems odd to me is that without any change to stddev I was able to make the `not_working` branch work by simply introducing a placeholder. So, the default value of `stddev` also works.

I'd suggest maybe others could try and repo this behaviour to see what's going on.

PS: The CTC "path not found" warnings are warnings and the training works with or without them.

**Comment by reuben** [comment]
FWIW, on my machine, using stddev=0.1 is the only way I can overfit the data. Without it, the loss is constantly infinite and we never improve.

**Comment by Cwiiis** [comment]
FWIW, this allows my machine to start converging using ldc93s1, previously it didn't ever seem to start converging at all.
